catalog/pipeline/fbc-release/0.12/tests/run.yaml

```diff
@@ -0,0 +1,38 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  name: fbc-release-run-empty-params
+spec:
+  params:
+    - name: snapshot
+      value: ""
+    - name: enterpriseContractPolicy
+      value: ""
+    - name: fromIndex
+      value: ""
+    - name: binaryImage
+      value: ""
+    - name: buildTags
+      value: ""
+    - name: addArches
+      value: ""
+    - name: requester
+      value: ""
+    - name: signingConfigMapName
+      value: ""
```

Comment:
where is the iibServiceConfigSecret?

Reply:
Removed it everywhere as it is out of scope of the ticket.